.drone.yml

```diff
@@ -1,16 +1,40 @@
 ---
 kind: pipeline
-name: default
+name: testing
 
 platform:
   os: linux
   arch: amd64
 
 steps:
+- name: vet
+  pull: always
+  image: golang:1.11
+  commands:
+  - go vet ./...
+  environment:
+    GO111MODULE: on
+  volumes:
+  - name: gopath
+    path: /go
+
 - name: test
+  pull: always
   image: golang:1.11
   commands:
-  - go test -v --cover ./...
+  - go test -cover ./...
+  environment:
+    GO111MODULE: on
+  volumes:
+  - name: gopath
+    path: /go
 
-...
+volumes:
+- name: gopath
+  temp: {}
 
+trigger:
+  ref:
+  - refs/heads/master
+  - "refs/tags/**"
+  - "refs/pull/**"
```

Comment:
this `trigger` block would result in feature branches being ignored

Reply:
Except you start a pull request.